Makefile

```diff
@@ -18,3 +18,5 @@ CMDS=csi-node-driver-registrar
 all: build
 
 include release-tools/build.make
+
+test: test-logcheck
```

Comment:
This will need more changes in the PR. As @bells17 pointed out, we should focus on the structured logging and not add `test-logcheck` here.

Reply:
I was mistaken. . I deleted the line.